dts_test_project/dts_test_project/settings.py

```diff
@@ -48,7 +48,16 @@
 
 TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 
-INSTALLED_APPS = list(set(TENANT_APPS + SHARED_APPS))
+INSTALLED_APPS = (
+    'dts_test_app',
+    'customers',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'tenant_schemas',
```

Comment:
Shouldn't it be in the first position? "When several applications provide management commands with the same name, Django loads the command from the application that comes first in INSTALLED_APPS."